Godeps/_workspace/src/k8s.io/kubernetes/pkg/proxy/userspace/loadbalancer.go

```diff
@@ -30,4 +30,5 @@ type LoadBalancer interface {
 	NextEndpoint(service proxy.ServicePortName, srcAddr net.Addr, sessionAffinityReset bool) (string, error)
```

Comment:
I want to believe this could be upstreamed as is, if you can cleanly decouple it and leave comments throughout the code about it.

Do you have an idling proposal that reflects this code that upstream has even seen?